.github/workflows/test.yml

```diff
@@ -0,0 +1,45 @@
+name: Tests
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Setup pip cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-py${{ matrix.python-version }}-pip--${{ hashFiles('requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-py${{ matrix.python-version }}-pip-
+            ${{ runner.os }}-py${{ matrix.python-version }}-
+
+      - name: Prepare environment
+        run: |
+          mkdir -p ~/.config/jd4
+          cp examples/config.yaml examples/langs.yaml ~/.config/jd4/
+          python -m pip install -r requirements.txt
+          python setup.py build_ext --inplace
+
+      - name: Unit test
+        run: python -m unittest -v jd4.case_test jd4.compare_test
+
+      - name: Build docker image
+        run: docker build -t vijos/jd4 .
```

Comment:
Can we push the docker image to the Docker registry? :)

Reply:
Currently, we are using auto-build service from Docker Hub. But it's also OK to push it here in the workflow

Reply:
The auto-build service is no longer free :-(